## <center>Практическое задание к лекции 4.

**1. Случайная непрерывная величина A имеет равномерное распределение на промежутке $(200, 800]$. Найдите ее среднее значение и дисперсию.**

In [258]:
mu=(800+200)/2
d=(800-200)**2/12

In [259]:
print(f'Среднее значение: М(А) =: {mu:.2f};\n'
      f'Дисперсия: D(A): {d:.2f}')

Среднее значение: М(А) =: 500.00;
Дисперсия: D(A): 30000.00


---  
#### Ответ:  
$M(A)=500.00\\D(A)=30000.00$  

---

**2. О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5? Если да, найдите ее.**

In [260]:
b=0.5+(12*0.2)**(1/2)
M=(b+0.5)/2

In [261]:
print(f'Левая граница распределения величины В - b = {b: .3f}\n'
      f'Среднее значение В на промежутке (0.5; {b: .3f}) M(B) = {M: .3f}')

Левая граница распределения величины В - b =  2.049
Среднее значение В на промежутке (0.5;  2.049) M(B) =  1.275


---
#### Ответ:  
Левая граница распределения: $b=2.049$  
Среднее значение: $M(B)=1.275$  

---

**3. Непрерывная случайная величина X распределена нормально и задана плотностью распределения $\displaystyle f(x)=\frac{1}{4\cdot\sqrt{2\pi}}\cdot e^{-\frac{(x+2)^2}{32}}$, Найдите:**  
   **а) $M(X)$**  
   **б) $D(X)$**  
   **в) $\displaystyle\sigma$ (среднее квадратичное отклонение)**

In [262]:
a=-2
std=4
print(f'M(x) = a = {a: .0f} \n'
      f'std(x) = {std: .0f} \n'
      f'D(x) = {std**2: .0f}'
     )

M(x) = a = -2 
std(x) =  4 
D(x) =  16


---  
#### Ответ:  
$M(X)=-2\\D(X)=16\\\displaystyle\sigma=4$  

---

**4. Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:**  
   **а) больше 182 см**  
   **б) больше 190 см**  
   **в) от 166 см до 190 см**  
   **г) от 166 см до 182 см**  
   **д) от 158 см до 190 см**  
   **е) не выше 150 см или не ниже 190 см**  
   **ё) не выше 150 см или не ниже 198 см**  
   **ж) ниже 166 см.** 

In [263]:
from scipy.stats import norm

In [264]:
mu=174
sigma=8

**а) больше 182 см**

In [265]:
pa=1-norm.cdf(182, mu, sigma)
print (f'P(X>=182)={pa: .4f} ')

P(X>=182)= 0.1587 


**б) больше 190 см** 

In [266]:
pb=1-norm.cdf(190, mu, sigma)
print (f'P(X>=190)={pb: .4f} ')

P(X>=190)= 0.0228 


**в) от 166 см до 190 см**

In [267]:
pv=norm.cdf(190, mu, sigma)-norm.cdf(166, mu, sigma)
print (f'P(166<=X<=190)={pv: .4f} ')

P(166<=X<=190)= 0.8186 


**г) от 166 см до 182 см** 

In [268]:
pg=norm.cdf(182, mu, sigma)-norm.cdf(166, mu, sigma)
print (f'P(166<=X<=182)={pg: .4f} ')

P(166<=X<=182)= 0.6827 


**д) от 158 см до 190 см** 

In [269]:
pd=norm.cdf(190, mu, sigma)-norm.cdf(158, mu, sigma)
print (f'P(158<=X<=190)={pg: .4f} ')

P(158<=X<=190)= 0.6827 


**е) не выше 150 см или не ниже 190 см** 

In [270]:
pe=norm.cdf(150, mu, sigma)+1-norm.cdf(190, mu, sigma)
print (f'P(150<=X U X>=190)={pe: .4f} ')

P(150<=X U X>=190)= 0.0241 


**ё) не выше 150 см или не ниже 198 см**

In [271]:
pe1=norm.cdf(150, mu, sigma)+1-norm.cdf(198, mu, sigma)
print (f'P(150<=X U X>=198)={pe1: .4f} ')

P(150<=X U X>=198)= 0.0027 


**ж) ниже 166 см.** 

In [272]:
pz=norm.cdf(166, mu, sigma)
print (f'P(X<=166)={pz: .4f} ')

P(X<=166)= 0.1587 



---

**5. На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?**

In [273]:
import math

In [274]:
std=math.sqrt(25)
sigma=(190-178)/std
print(f'Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, \n'
      f'на {sigma} сигм.')

Рост человека, равный 190 см, отклоняется от математического ожидания роста в популяции, 
на 2.4 сигм.


---  
#### Ответ:  
на $2.4$ сигмы  

---